src/__tests__/home.test.js

```diff
@@ -0,0 +1,8 @@
+import renderer from 'react-test-renderer';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Kindly refrain from using `eslint disable`, as that defies the purpose of using linters in the first place.